### ISSUE TITLE
Accessibility to allow blind organists to use GrandOrgue

### ISSUE THREAD

**Issue by lewisalexander2020** [GrandOrgue/grandorgue#1552]
Hi there, 

I'm  writing this as a blind organist, on a mac using VoiceOver. I am also a developer behind a speaking digital organ solution and whilst working with a particular software developer I cannot disclose for legal reasons, I wanted to investigate accessibility support for GrandOrgue in MacOS.

Initial setup allows me to navigate menus and setup resources, but the main organ console panel has no degree of support what so ever, meaning that VoiceOver doesn't even see the window nor have any ability to navigate it, interact with any elements, etc.

I'd like to help where I can with advice and testing, but I don't know what framework this is built on nor what resources you're using to develop this nor create the libraries / GUI's for the libraries.

The reason behind this request is that, other than a company I now work for, supporting their system and development, other organ software such as Milan Digital Audio's Hauptwerk, has never been and to this day, is not accessible to blind organists, there is some workaround for windows users but that is messy and unstable.. I've made some efforts over the years to try and persuade the developers behind Hauptwerk to find solutions to make their software VoiceOver friendly, but it's either been ignored or promises offered and broken, really sad when someone does this and fails on it.

GrandOrgue is a great system, if we could make it so that VoiceOver can work with the main features such as the main console view and other areas, then we can make this an available and supported option for blind and sight impaired organists. If any of the developers behind this project would like to get in touch with me about this, I'd really appreciate hearing from you.

The last time I tried using this was a couple of years back and it's evident it's moved on so much and seems better to set up than it did. keep up the amazing work. If we could make this app work so that VoiceOver can speak the contents of the console window, interact with buttons, etc, edit functions, etc, then this would be a great solution and you'd have me to support this with video tutorials and demonstrations if that is of interest.

All the best.

Lew
